charts/kargo/templates/common/cert-issuer.yaml

```diff
@@ -1,5 +1,5 @@
 
-{{- if or (and .Values.api.enabled .Values.api.ingress.enabled .Values.api.ingress.tls.enabled .Values.api.ingress.tls.selfSignedCert) (and .Values.webhooksServer.enabled .Values.webhooksServer.tls.selfSignedCert) (and .Values.api.enabled .Values.api.oidc.enabled .Values.api.oidc.dex.enabled .Values.api.oidc.dex.tls.selfSignedCert) }}
+{{- if or (and .Values.api.enabled .Values.api.tls.enabled .Values.api.tls.selfSignedCert) (and .Values.api.enabled .Values.api.ingress.enabled .Values.api.ingress.tls.enabled .Values.api.ingress.tls.selfSignedCert) (and .Values.api.enabled .Values.api.oidc.enabled .Values.api.oidc.dex.enabled .Values.api.oidc.dex.tls.selfSignedCert) (and .Values.externalWebhookServer.enabled .Values.externalWebhookServer.tls.enabled .Values.externalWebhookServer.tls.selfSignedCert) (and .Values.externalWebhookServer.enabled .Values.externalWebhookServer.ingress.enabled .Values.externalWebhookServer.ingress.tls.enabled .Values.externalWebhookServer.ingress.tls.selfSignedCert) (and .Values.webhooksServer.enabled .Values.webhooksServer.tls.selfSignedCert) }}
```

Comment:
Apart from enabling the cert issuer in the event of new external-webhook-server-related scenarios, there's a small bug fix in here...

`(and .Values.api.enabled .Values.api.tls.enabled .Values.api.tls.selfSignedCert)` was previously unaccounted for.